master/pkg/tasks/copy.go

```diff
@@ -22,7 +24,8 @@ const (
 
 func harnessArchive(harnessPath string, aug *model.AgentUserGroup) container.RunArchive {
 	var harnessFiles archive.Archive
-	wheelPaths, err := filepath.Glob(filepath.Join(harnessPath, "*.whl"))
+	validWhlNames := fmt.Sprintf("*%s*.whl", version.Version)
+	wheelPaths, err := filepath.Glob(filepath.Join(harnessPath, validWhlNames))
```

Comment:
Do we expect the `wheelPaths` to be predictable / something we should check?

Reply:
As far as I understand, it should always at `$(ROOT)/wheels/determined*$(VERSION)*.whl` where ROOT is `root` from `master.yml` and `version` is the version that is set by `ldflags` flag to `go build` when we build the master. I guess these paths could be even more specific.

Reply:
The names themselves are standardized by the PEP for whl's, https://www.python.org/dev/peps/pep-0427/#file-name-convention. (I didn't know this, was curious)

Reply:
Sorry, I mean `len(wheelPaths)` -- i.e., do we expect to find exactly 1 and should we check for that?

Reply:
Yeah, I think it should always be 3 (determined, determined-cli and determined-common should be in `$ROOT/wheels`).